_posts/en/newsletters/2020-08-12-newsletter.md

```diff
@@ -0,0 +1,215 @@
+
+  q4="Why is Tor v3 important for Bitcoin?"
+  a4="Tor v2 is planned to be [deprecated on Sept 15, 2020][19031 v2 deprecate]
+      and obsolete on July 15, 2021 ([schedule][19031 v2 schedule]). Tor v3
+      provides much better security and privacy. It uses longer, 56-character
+      addresses and therefore requires implementation of BIP155 `addrv2`."
+  a4link="https://bitcoincore.reviews/19031#l-203"
+
+  q5="Will we need another upgrade, aka `addrv3`, in the near future?"
+  a5="Not for some time. The only `addrv2` constraint is that addresses be
+      stored in not more than 512 bytes. Any future format within that size,
+      including variable-length route descriptions, should be supported."
+  a5link="https://bitcoincore.reviews/19031#l-253"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.11.0-beta.rc2][lnd 0.11.0-beta] is a release candidate for the
+  next major version of this project.  It allows accepting [large
+  channels][topic large channels] (by default, this is off) and contains
+  numerous improvements to its backend features that may be of interest
+  to advanced users (see the [release notes][lnd 0.11.0-beta]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #18991][] changes the way that Bitcoin Core responds to
+  `getaddr` requests from its peers. Bitcoin nodes learn about potential new
+  peers to connect to in two different ways. Firstly, each node periodically
+  announces its own network address to its peers. Those peers re-announce the
+  address to some of their own peers, who re-announce to their peers, and so on,
+  propagating the address around the network. Secondly, a node can explicitly
+  request addresses from a peer using a `getaddr` message. To prevent a peer
+  from learning about all of the addresses in its address manager, Bitcoin Core
+  only provides a subset of those addresses when a peer sends it a `getaddr`
+  message, and only responds to a single `getaddr` message from each peer.
+  However, an adversary may still be able to learn all the addresses a peer knows
+  by making multiple connections to that peer and sending `getaddr` messages on
+  each connection. The adversary could potentially use that information to
+  fingerprint the peer or infer network topology. To prevent that, this PR
+  changes Bitcoin Core's behaviour to cache responses to `getaddr` requests and
+  provide the same response to all peers who request addresses in a 24 hour
+  window.
```

Comment:
Great description @jnewbery! I wonder if it would be good to mention that it's a rolling window (randomly varying between 21 and 27 hours, if memory serves).

Reply:
good idea. Added the word "rolling"